doc/install.rst

```diff
@@ -32,16 +32,18 @@ PyGMT requires Generic Mapping Tools (GMT) version 6 as a minimum, which is the
 released version that can be found at
 `this website <https://www.generic-mapping-tools.org>`__.
 
-We need the very latest GMT since there are many changes being made to GMT itself in
+We need the latest GMT since there are many changes being made to GMT itself in
 response to the development of PyGMT, mainly the new
-`modern execution mode <https://gmt.soest.hawaii.edu/projects/gmt/wiki/Modernization>`__.
+`modern execution mode <https://docs.generic-mapping-tools.org/latest/cookbook/introduction.html#modern-and-classic-mode>`__.
 
-Compiled conda packages of GMT for Linux, Mac and Windows are provided through
+Compiled conda packages of GMT for Linux and Mac are provided through
 `conda-forge <https://anaconda.org/conda-forge/gmt>`__.
 Advanced users can also
 `build GMT from source <https://github.com/GenericMappingTools/gmt/blob/master/BUILDING.md>`__
-instead, which is not so recommended but we would love to get feedback from anyone who
-tries.
+instead, which is not so recommended but we would love to get feedback from anyone who tries.
+For Windows, conda GMT packages are available, but they do not currently work with PyGMT,
+so users will need to build from source or use the Windows Subsystem for Linux, see
+`here <https://github.com/GenericMappingTools/pygmt/pull/313>`__ for more details.
```

Comment:
I'm wondering if it's time we removed the GMT 6 install instructions from here. It's a workaround that was necessary but might not be any more. There shouldn't be problems installing PyGMT on the base conda environment. 

Maybe instead of removing, we could move it further down and label it "If you have trouble install GMT 6" or something like that. What do you think?

Reply:
It's probably better to keep it near the beginning for now, just so users (especially Windows ones) are aware of potential problems up front. The conda-forge `pygmt` package might only work for Linux/MacOS initially, and we need to remember that WIndows still takes up ~90% of the world's operating system market share. It'll be more frustrating for someone to have gone through installing `gmt` and `pygmt` from conda-forge and realize that it doesn't work on WIndows because they missed the fine print at the bottom.